SUMMARY.md

```diff
@@ -199,11 +202,11 @@
 
 ## Client App SDKs
 
-* [Javascript SDK Reference](https://authgear.github.io/authgear-sdk-js/docs/)
-* [iOS SDK Reference](https://authgear.github.io/authgear-sdk-ios/)
-* [Android SDK Reference](https://authgear.github.io/authgear-sdk-android/)
-* [Flutter SDK Reference](https://authgear.github.io/authgear-sdk-flutter/)
-* [Xamarin SDK Reference](https://authgear.github.io/authgear-sdk-xamarin/)
+* [Javascript SDK api-reference](https://authgear.github.io/authgear-sdk-js/docs/)
+* [iOS SDK api-reference](https://authgear.github.io/authgear-sdk-ios/)
+* [Android SDK api-reference](https://authgear.github.io/authgear-sdk-android/)
+* [Flutter SDK api-reference](https://authgear.github.io/authgear-sdk-flutter/)
+* [Xamarin SDK api-reference](https://authgear.github.io/authgear-sdk-xamarin/)
```

Comment:
The "Reference" in Lines 205-209 should not be renamed to "api-reference". They should be "JavaScript SDK Reference", etc.